src/java.base/share/classes/javax/security/auth/callback/CallbackHandler.java

```diff
@@ -113,7 +113,7 @@ public interface CallbackHandler {
      *          System.err.print(nc.getPrompt());
      *          System.err.flush();
      *          nc.setName((new BufferedReader
-     *                  (new InputStreamReader(System.in))).readLine());
+     *                  (new InputStreamReader(System.in, "stdin.encoding"))).readLine());
```

Comment:
Needs to be `System.getProperty("stdin.encoding"))`.

Reply:
Good catch! Thanks.